pages/movies/[id].js

```diff
@@ -75,38 +87,84 @@ const MovieImageContainer = styled.div`
   width: 100vw;
   height: 50vw;
 `;
+const StyledParagraph = styled.p`
+  margin: 0;
+`;
 
 const GenreList = styled.ul`
   padding: 0px;
+
   display: flex;
-  justify-content: space-around;
+
   position: relative;
   list-style: none;
+  gap: 10px;
 `;
 
-const Button = styled.button`
+const Button = styled(Link)`
   position: relative;
-  width: 100%;
-  height: 20px;
-  background-color: grey;
+  margin-left: -20px;
+  margin-top: -15px;
+  transform: skew(45deg);
+  width: 25%;
+  height: 30px;
+  border-style: none;
+  z-index: 5;
+  padding-left: 25px;
+  display: inline-block;
+  text-decoration: none;
+  box-shadow: rgba(0, 0, 0, 0.3) 1px 1px 5px 0px;
 
   ${({ variant }) =>
     variant === "remove" &&
     css`
-      background-color: hotpink;
+      background-color: var(--fourth-color);
+      color: #e97f7f;
     `}
 
   ${({ variant }) =>
     variant === "add" &&
     css`
-      background-color: grey;
+      background-color: var(--third-color);
+      color: #7aa2cb;
     `}
 `;
 
-const WrapButton = styled.div`
-  @media (min-width: 1024px) {
-    &:hover ${Button} {
-      background-color: hotpink;
-    }
-  }
+const DivContainer = styled.div`
+  position: relative;
+  margin-top: 0px;
+  width: 100%;
+  height: 25px;
+  background-color: var(--secondary-color);
+  z-index: 0;
+`;
```

Comment:
`z-index: 0` is the default and therefore is not needed here.
```suggestion
const DivContainer = styled.div`
  margin-top: 0px;
  width: 100%;
  height: 25px;
  background-color: var(--secondary-color);
`;
```